core/breakpoints/manager.php

```diff
@@ -0,0 +1,330 @@
+<?php
+namespace Elementor\Core\Breakpoints;
+
+use Elementor\Core\Base\Module;
+use Elementor\Core\Kits\Documents\Tabs\Settings_Layout;
+use Elementor\Core\Responsive\Files\Frontend;
+use Elementor\Core\Responsive\Responsive;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Manager extends Module {
+
+	const BREAKPOINT_OPTION_PREFIX = 'viewport_';
+	const BREAKPOINT_KEY_MOBILE = 'mobile';
+	const BREAKPOINT_KEY_MOBILE_EXTRA = 'mobile_extra';
+	const BREAKPOINT_KEY_TABLET = 'tablet';
+	const BREAKPOINT_KEY_TABLET_EXTRA = 'tablet_extra';
+	const BREAKPOINT_KEY_LAPTOP = 'laptop';
+	const BREAKPOINT_KEY_DESKTOP = 'desktop';
```

Comment:
remove desktop breakpoint and change `Stylesheet` class behavior 'accordingly'

Reply:
Done.